packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React, { PropTypes } from 'react';
```

Comment:
Once Matt's PR for React 15.5 compatibility is merged in, PropTypes will need imported from the prop-types package. Should this be done before his is merged in so there are less changes in the future?